src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java

```diff
@@ -13,4 +13,10 @@ public class Book {
     int id;
```

Comment:
Modify it to have long, error created by @Mayurp375

Reply:
resolved sir